tests/catboost/test_catboost_model_export.py

```diff
@@ -419,3 +420,11 @@ def test_pyfunc_serve_and_score_sklearn(reg_model):
     )
     scores = pd.read_json(resp.content.decode("utf-8"), orient="records").values.squeeze()
     np.testing.assert_array_almost_equal(scores, model.predict(inference_dataframe.head(3)))
+
+
+def test_save_model_with_code_paths(cb_model, model_path):
```

Comment:
Love the unit tests! Can we add `log_model` to the call as well? I know that it always calls `save_model` under the hood, but that seems like something the unit tests shouldn't know